conjure-java-core/src/integrationInput/java/com/palantir/product/DialogueDialogueSampleService.java

```diff
@@ -0,0 +1,522 @@
+package com.palantir.product;
+
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.google.common.base.Preconditions;
+import com.google.common.collect.ImmutableList;
+import com.google.common.util.concurrent.ListenableFuture;
+import com.palantir.conjure.java.serialization.ObjectMappers;
+import com.palantir.dialogue.Call;
+import com.palantir.dialogue.Calls;
+import com.palantir.dialogue.Channel;
+import com.palantir.dialogue.Deserializer;
+import com.palantir.dialogue.Deserializers;
+import com.palantir.dialogue.DialogueOkHttpErrorDecoder;
+import com.palantir.dialogue.Endpoint;
```

Comment:
off-topic: I see `Endpoint`  to designate a single route. Same for conjure (under `EndpointName`). Whereas in undertow-lib, we use it to designate something that is gonna register many different routes. I think this is a bit confusing. Could we potentially rename `Endpoint` to `Endpoints` in undertow-lib @cakofony ?